pkg/sql/sem/tree/overload.go

```diff
@@ -67,8 +67,19 @@ type Overload struct {
 
 	AggregateFunc func([]*types.T, *EvalContext, Datums) AggregateFunc
 	WindowFunc    func([]*types.T, *EvalContext) WindowFunc
-	Fn            func(*EvalContext, Datums) (Datum, error)
-	Generator     GeneratorFactory
+
+	// Only one of the following three attributes can be set.
```

Comment:
I think asserting this in the `init` function in `all_builtins.go` would be nice.